packages/interfaces/src/index.ts

```diff
@@ -187,6 +189,7 @@ export interface DecodedMessage {
   contentTopic: string | undefined;
   timestamp: Date | undefined;
   rateLimitProof: RateLimitProof | undefined;
+  ephemeral?: boolean;
```

Comment:
```suggestion
  ephemeral: boolean;
```

The field must be present, that's why we don't use `?` in this interface.